tests/functional_tests/ffront_tests/past_common_fixtures.py

```diff
@@ -41,6 +41,14 @@ def identity(in_field: Field[[IDim], "float64"]) -> Field[[IDim], "float64"]:
     return identity
 
 
+@pytest.fixture
+def empty_identity_def():
+    def empty_identity(empty_field: Field[[], "float64"]):
+        return empty_field
+
+    return empty_identity
+
+
```

Comment:
```suggestion
```
Looks like dead-code by now.

Reply:
I left this because it may be useful in the future